canister/src/state.rs

```diff
@@ -50,6 +50,9 @@ pub struct State {
     /// Flag to control access to the apis provided by the canister.
     #[serde(default)]
     pub api_access: Flag,
+
+    /// Flag to control syncing of the next blocks.
```

Comment:
I don't think this is correct. This flag doesn't effect the syncing of the blocks.

```suggestion
    /// Flag to determine if the API should be automatically disabled if the canister isn't fully synced.
```